web/impact/impact/utils.py

```diff
@@ -12,6 +13,15 @@
         "2010-01-01T00:00:00Z",
         "%Y-%m-%dT%H:%M:%SZ"))  # format based on browsable API output
 
+UPDATE_STATEMENT = "UPDATE {table} SET updated_at = %s WHERE id = %s"
+
+
+def override_updated_at(model, value):
```

Comment:
interesting - as long as this is here only for tests. I think we should leave the actual field as readonly